pom.xml

```diff
@@ -70,6 +70,37 @@
                     <autoReleaseAfterClose>false</autoReleaseAfterClose>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>com.coveo</groupId>
+                <artifactId>fmt-maven-plugin</artifactId>
+                <version>2.10</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                        <phase>process-sources</phase>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>au.com.acegi</groupId>
+                <artifactId>xml-format-maven-plugin</artifactId>
```

Comment:
This formatter seems too strict to me. For example, it does not allow multi-line XML tags, which are often convenient and more readable. For example, for OSGi:
```
            <Import-Package>
              org.osgi.*,
              io.stargate.db.*,
              org.apache.cassandra.stargate.db.*,
              !*
            </Import-Package>
```
vs.
```
            <Import-Package>org.osgi.*, io.stargate.db.*, org.apache.cassandra.stargate.db.*, !*</Import-Package>
```

Reply:
seems to work with `CDATA`:
```
            <Import-Package><![CDATA[
              org.osgi.*,
              io.stargate.db.*,
              org.apache.cassandra.stargate.db.*,
              !*
            ]]></Import-Package>
```